### PR TITLE
WIB Improvement/safety structure

### PR DESCRIPTION
**Pull request by TimBuckers**

### CODE REVIEW

File: march_safety/include/march_safety/InputDeviceSafety.h [hunk 1]
```diff
@@ -20,14 +22,11 @@ class InputDeviceSafety
 
   void inputDeviceAliveCallback(const std_msgs::TimeConstPtr& msg);
 
-  march_shared_resources::Error createErrorMessage();
-
-  march_shared_resources::Error createFutureErrorMessage();
-
 public:
-  InputDeviceSafety(ros::Publisher* error_publisher, ros::NodeHandle n);
+  InputDeviceSafety(ros::NodeHandle* n, SafetyHandler* safety_handler);
```
Comment (Ishadijcks) [inline march_safety/include/march_safety/InputDeviceSafety.h]: Does this still need a nodehandle?

File: march_safety/src/SafetyNode.cpp [hunk 2]
```diff
@@ -20,19 +22,25 @@ int main(int argc, char** argv)
   ros::Rate rate(200);
 
   // Create an error publisher to notify the system (state machine) if something is wrong
-  ros::Publisher error_publisher = n.advertise<march_shared_resources::Error>(TopicNames::error, 1000);
+  ros::Publisher error_publisher = n.advertise<march_shared_resources::Error>("march/error", 1000);
```
Comment (Ishadijcks) [inline march_safety/src/SafetyNode.cpp]:
```suggestion
  ros::Publisher error_publisher = n.advertise<march_shared_resources::Error>("/march/error", 1000);
```

File: march_shared_resources/msg/Error.msg [hunk 3]
```diff
@@ -1,10 +1,8 @@
-# The error code
-int16 error_code
-
 # Readable message that describes the error
 string error_message
 
 # The type of the error, the type is used to determine how to handle the error
 uint8 type
 uint8 FATAL             = 0     # The error indicates a serious malfunctioning of the system this is unrecoverable
+uint8 NON_FATAL             = 1     # The error indicates a serious malfunctioning of the system this is unrecoverable
```
Comment (Ishadijcks) [inline march_shared_resources/msg/Error.msg]: Update comment

File: march_safety/include/march_safety/InputDeviceSafety.h (hunk 1 above)
Comment (TimBuckers) [reply march_safety/include/march_safety/InputDeviceSafety.h]: Yes

**Pull request merged**